apps/admin-gui/src/assets/i18n/en.json

```
@@ -1355,12 +1355,24 @@
     },
     "CHANGE_STATUS": {
       "TITLE": "Change status",
-      "CANCEL": "Cancel",
-      "CURRENT_STATUS": "Current status",
-      "NEW_STATUS": "New status",
-      "CHANGE_VALID": "Changing status to VALID will trigger automatic configuration for provided resources. \nIf successful, member will have access on provided resources. \nIf not, see displayed error message and do manual configuration on 'settings' tab on members detail.",
+      "CURRENT_STATUS": "Current status: ",
+      "VALID_DESCRIPTION": "Member is properly configured and have access on provided resources.",
       "INVALID_DESCRIPTION": "Member have configuration error and DON'T have access on provided resources. You can check what is wrong by changing member's status to VALID. If possible, procedure will configure all necessary settings by itself.",
-      "SUCCESS": "Member status was changed"
+      "EXPIRED_DESCRIPTION": "Member didn't extend membership and DON'T have access on provided resources.",
```

This might be a bit confusing but some services allow also expired members. This solely depends on service implementation. Some description like this might be more accurrate.
```suggestion
      "EXPIRED_DESCRIPTION": "Member didn't extend membership and might NOT have access to all services.",
```